include/mbedtls/timing.h

```diff
@@ -90,6 +95,16 @@ void mbedtls_timing_set_delay( void *data, uint32_t int_ms, uint32_t fin_ms );
  */
 int mbedtls_timing_get_delay( void *data );
 
+/**
+ * \brief          Get the final timing delay
+ *
+ * \param data     Pointer to timing data
+ *                 Must point to a valid \c mbedtls_timing_delay_context struct.
+ *
+ * \return         Final timing delay in milliseconds.
+ */
+uint32_t mbedtls_timing_get_final_delay( void *data );
```

Comment:
Question: why make the argument `void *` rather than `mbedtls_timing_delay_context *` - or actually `const mbedtls_timing_delay_context *`?

Reply:
Ah, I think I got confused when merging the fixes for builds without `MBEDTLS_HAVE_TIME` defined. I thought that if `MBEDTLS_HAVE_TIME` was not defined, neither would `mbedtls_timing_delay_context` be, hence the need for a `void *`. I now see this is wrong.